src/services/users/users.ts

```diff
@@ -14,6 +15,7 @@ export const createAccount = async (
   account_data: CreateAccountRequest,
```

Comment:
This does not align with what the backend expects. This causes an error in user creation. The current version of development has a working version.